tpm/pcrs.go

```diff
@@ -156,3 +156,17 @@ func newPCRInfo(rw io.ReadWriter, pcrNums []int) (*pcrInfo, error) {
 
 	return pcri, nil
 }
+
+// CustomPCRInfoLong creates and returns a PcrInfoLong structure for the given // PCR and hashes.
+func CustomPCRInfoLong(location byte, pcrs map[int][]byte) (*PcrInfoLong, error) {
```

Comment:
The name "CustomPCR..." makes me think that these are PCRs with index values that can't otherwise be accessed. However, it looks to me like these are actually pre-specified hashes for known PCR index values (rather than reading the values of the specified PCRs).

This looks to me like the Go equivalent of a constructor for the PCRInfoLong type, and Go style usually gives those names starting with New. So, this could be NewPCRInfoLongWithHashes.

However, I'm open to other names as well. I'd just like to get away from the connotations of "Custom".

Reply:
Sounds good to me, done